app/models/user.rb

```diff
@@ -1,2 +1,5 @@
 class User < ApplicationRecord
+  def self.greet
```

Comment:
Sensitive data such as "access_token" and "auth_token" should not be stored in plain text in the database. This exposes the data to potential breaches. To mitigate this, you should encrypt these fields before storing them in the database.

```suggestion
create_table "users", force: :cascade do |t|
    t.string "dob"
    t.integer "age"
    t.string "email_address"
    t.string "email"
    t.string "access_token", encrypt: true
    t.string "auth_token", encrypt: true
    t.string "full_name"
    t.datetime "created_at", precision: 6, null: false
    t.datetime "updated_at", precision: 6, null: false
end
```
Please note that this suggestion assumes that you have a mechanism in place to handle the encryption and decryption of these fields. If not, you will need to implement this functionality.